.github/workflows/pull-request-test.yml

```diff
@@ -126,46 +128,55 @@ jobs:
           test-timeout-mins: "120"
 
   test-pr-cypress:
-    runs-on: ubuntu-20.04
     if: ${{ github.event.action != 'closed' &&
       needs.check-deploy.outputs.pr-contains-string == 'true' &&
       (needs.check-deploy.outputs.test-enabled == 'true' ||
       needs.check-deploy.outputs.test-cypress-enabled == 'true') }}
     needs: [check-deploy, deploy-pr]
+    runs-on: ubuntu-22.04
+
+    strategy:
+      fail-fast: false
+      max-parallel: 1
```

Comment:
```suggestion
```

It seems that parallel execution works, see: https://github.com/SwissDataScienceCenter/renku/pull/3166.

Reply:
Oh... I tried that quite a few times and I kept getting 401 errors on the user endpoint.
Can we try to re-run it a few times from scratch? I fear the issue happens when one of the tests fails early or the logout in the publicProject spec runs before the others, but that's not always the case

Reply:
Here is an example with the `401: Unauthorized` error on the `GET /user` endpoint that we get when running the tests in parallel
https://github.com/SwissDataScienceCenter/renku/actions/runs/5589204312/jobs/10217824415